doc/translations/zh_CN.po

```diff
@@ -43523,7 +43523,7 @@ msgid ""
 "glTF 2.0 3D import pipeline. This importer requires Blender to be installed "
 "by the user, so that it can be used to export the scene as glTF 2.0.\n"
 "The location of the Blender binary is set via the [code]filesystem/import/"
-"blender/blender3_path[/code] editor setting.\n"
+"blender/blender_path[/code] editor setting.\n"
```

Comment:
Translations should not be edited here, `.po` files are generated from Weblate.

Reply:
Thanks for the headsup, I'll remove those changes from the PR